web/src/main/java/org/akaza/openclinica/controller/openrosa/OpenRosaSubmissionController.java

```diff
@@ -251,7 +251,9 @@
 
         int count = 0;
         for (EventCrf evCrf : eventCrfs) {
-            if (evCrf.getStatusId() == org.akaza.openclinica.domain.Status.UNAVAILABLE.getCode()) {
+            if (evCrf.getStatusId() == org.akaza.openclinica.domain.Status.UNAVAILABLE.getCode()
```

Comment:
Would be useful to import this org.akaza.openclinica.domain.Status statically so we don't use long names